### PR TITLE
nuke the docs

### PR DESCRIPTION
**Pull request by andymckay**

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -5,249 +5,20 @@ Welcome to Olympia's documentation!
...
+   :maxdepth: 2
 
+   topics/api/index
    topics/install-olympia/index
-   topics/hacking/index
+   topics/development/index
 
 .. toctree::
    :maxdepth: 2
    :glob:
```
Comment (magopian) [inline docs/index.rst]: Is this still needed then?

File: docs/topics/install-olympia/index.rst [hunk 2]
```diff
@@ -1,16 +1,14 @@
 ===============
-Install Olympia
+Server Install
 ===============
 
+
+
 This is a legacy section for installing Olympia manually (and painfully) by hand.
 **Instead, you'll find it easier to** :ref:`install Olympia from Docker <install-with-docker>`.
```
Comment (magopian) [inline docs/topics/install-olympia/index.rst]: I'm not sure this sentence is still relevant: it present two ways to install the server, using docker (the first method, the one you get when you click "next"), and the legacy one after that.

Maybe we could have something slightly different: remove the above sentence (or reword it to explain that the recommended way to install is via docker), and then explain below the toctree (that would only have the "docker" install) that there's also a legacy deprecated way to install it, and then link to `deprecated/index` there (without a toctree in this page to avoid confusion).

**Comment by magopian** [comment]
I see there's a few docs that are simply gone. Is there a place you plan to re-add them? The OAuth one might not be needed in the future when we have FxA everywhere, but until then? Aren't the others like "mobile", "templates", "translations"... useful?

r+wc

**Comment by andymckay** [comment]
> On Nov 2, 2015, at 1:41 AM, Mathieu Agopian <notifications@github.com> wrote:
> 
> I see there's a few docs that are simply gone. Is there a place you plan to re-add them? The OAuth one might not be needed in the future when we have FxA everywhere, but until then? Aren't the others like "mobile", "templates", "translations"... useful?
> 
> 

Useful to whom? Quite frankly I couldn’t see anything at all useful in them.

File: docs/topics/install-olympia/index.rst (hunk 2 above)
Comment (andymckay) [inline docs/topics/install-olympia/index.rst]: Thanks, fixed up.

File: docs/index.rst (hunk 1 above)
Comment (andymckay) [inline docs/index.rst]: nope, thanks
